### ISSUE TITLE
Wrong include in the swift-winui projections.json

### ISSUE THREAD

**Issue by rayman-zhao** [thebrowsercompany/swift-winrt#188]
Hi, does the "Microsoft.UI.ColorHelper/Colors" in the projections.json need to be in swift-windowsappsdk ? It does not export these symbols.

**Comment by tristanlabelle** [comment]
You're right! Those repos are stale though so I have archived them. Let us know if we can help with using this project directly.

**Comment by rayman-zhao** [comment]
Thank you and the wonderful tools! We are actively trying to use Swift to develop a Windows native software based on WinAppSDK. The swift-winrt is much better and easier than CppWinRT. These preconfigured  swift-* libs really helpful, specifically for beginners. A good designed program template will share your experiences, and give developers clear guidance. So it's worth to maintain and very useful.

Microsoft should support you guys, to include swift-winrt in the WinAppSDK ecosystem. Apple should also support you guys, to write AppleMusic etc. Windows apps with Swift and swift-winrt. So that desktop developer will have much more confidence to use Swift on other platforms.

**Comment by tristanlabelle** [comment]
Thanks for the kind words! The pregenerated repos are useful for experimenting but they won't work for real-world app because they bake in a dependency on a specific version of the Windows SDK and Windows App SDK which are now outdated, and they also bake in the specific subset of types for which projections are generated, all of which might not match what you need in your project. We cannot generate those repos in a way that will work for every user. This is why we really recommend using swift-winrt directly, which is what we do for Arc, as it allows you to control those parameters.
